### DISCUSSION TITLE
Shopify / Magento Integration for model-viewer

### DISCUSSION THREAD

**Discussion by Jasonp8000** [google/model-viewer#3559] [General]
Hi everyone,

I understand this topic may be out of scope however, I though I would reach out. I'm looking into Integrating model-viewer into Shopify, Magento and woocommerce. If anyone could point me in the right direction it would be much appreciated. 

I sure many people in the community would benefit from this. 

 
**Shopify - Angle 3D (They use model-viewer)**
https://angle-3d-demo.myshopify.com/collections/frontpage/products/angle-3d-leather-bag?tdaState=eyJscmVnaSI6IjlhbXJwcCIsIm04MzFqOCI6InZhNWg0aSIsInU0cWRoYSI6InR2d3hqIiwiOHowc3o0bSI6Imp2dXE0NCIsInhjbnI5dyI6IngwMTNkNyIsInJoYnRkIjoiYnRwaXEifQ%253D%253D 

Magento - https://store.webkul.com/magento2-webar-product-image.html#hosting

**Comment by elalish** [comment]
I think both of those links use model-viewer; can you be a bit more specific as to what question you're asking? Shopify even has model-viewer built into one of their in-house themes.
